static/include/services/apis/generated/slam.md

```diff
@@ -0,0 +1,357 @@
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/slam/client/index.html#viam.services.slam.client.SLAMClient.get_internal_state).
+
+```python {class="line-numbers linkable-line-numbers"}
+slam = SLAMClient.from_robot(robot=robot, name="my_slam_service")
+
+# Get the internal state of the SLAM algorithm required to continue mapping/localization.
+internal_state = await slam.get_internal_state()
+```
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+**Parameters:**
+
+- `name` [(String)](https://api.flutter.dev/flutter/dart-core/String-class.html) (required):
+
+**Returns:**
+
+- None.
+
+For more information, see the [Flutter SDK Docs](https://flutter.viam.dev/viam_protos.service.slam/SLAMServiceClient/getInternalState.html).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetProperties
+
+Get information about the current SLAM session.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `timeout` ([float](https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex)) (optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- (viam.services.slam.slam.SLAM.Properties): The properties of SLAM
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/slam/client/index.html#viam.services.slam.client.SLAMClient.get_properties).
+
+```python {class="line-numbers linkable-line-numbers"}
+slam_svc = SLAMClient.from_robot(robot=robot, name="my_slam_service")
+
+# Get the properties of your current SLAM session.
+slam_properties = await slam_svc.get_properties()
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [(Properties)](https://pkg.go.dev#Properties):
+- [(error)](https://pkg.go.dev/builtin#error):
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/slam#Service).
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+**Parameters:**
+
+- `name` [(String)](https://api.flutter.dev/flutter/dart-core/String-class.html) (required):
+
+**Returns:**
+
+- None.
+
+For more information, see the [Flutter SDK Docs](https://flutter.viam.dev/viam_protos.service.slam/SLAMServiceClient/getProperties.html).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### InternalStateFull
+
+`InternalStateFull` concatenates the streaming responses from `InternalState` into the internal serialized state of the SLAM algorithm.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- [(Service))](<INSERT PARAM TYPE LINK>):
```

Comment:
When do the param type links get inserted? Is that part of what I'm meant to ignore for now?

Reply:
Yes, for now this is the part to ignore. Missing are:
- `slamSvc` [Service](https://pkg.go.dev/go.viam.com/rdk/services/slam#Service)
- `returnEditedMap` [bool](https://pkg.go.dev/builtin#bool)

I have https://viam.atlassian.net/browse/DOCS-2363 to fix these separately.